playbooks/templates/artifact/index.html

```diff
@@ -399,7 +399,7 @@ <h3>Browse Artifacts</h3>
             {
               icon: 'graphic_eq',
               title: 'Build Summary Dashboard',
-              url: "http://rpc-repo.rackspace.com/rpcgating/buildsummary/"
+              url: "http://rpc-repo.rackspace.com/rpcgating/buildsummary/index.html#/job/" + this.job_name
```

Comment:
Doesn't really improve much on this line, but it'd be interesting to see if line 435 could be changed to:

```
url: `${this.container_public_url}/${this.archive_base_name}/${files[0].path}`
```

Reply:
Yeah I agree a template string would be cleaner there.